docs/src/user-guide/command-line-interface.md

```diff
@@ -314,53 +390,68 @@ npx eslint --ignore-path .gitignore file.js
 
 #### `--no-ignore`
 
-Disables excluding of files from `.eslintignore`, `--ignore-path`, `--ignore-pattern`, and `ignorePatterns` property in config files.
+Disables excluding of files from `.eslintignore` files, `--ignore-path` flags, `--ignore-pattern` flags, and the `ignorePatterns` property in config files.
 
-Example:
+* **Argument Type**: No argument.
+
+##### `--no-ignore` example
 
 ```shell
 npx eslint --no-ignore file.js
 ```
 
 #### `--ignore-pattern`
 
-This option allows you to specify patterns of files to ignore (in addition to those in `.eslintignore`). You can repeat the option to provide multiple patterns. The supported syntax is the same as for `.eslintignore` [files](configuring/ignoring-code#the-eslintignore-file), which use the same patterns as the `.gitignore` [specification](https://git-scm.com/docs/gitignore). You should quote your patterns in order to avoid shell interpretation of glob patterns.
+This option allows you to specify patterns of files to ignore (in addition to those in `.eslintignore`).
+
+* **Argument Type**: String. The supported syntax is the same as for [`.eslintignore` files](configuring/ignoring-code#the-eslintignore-file), which use the same patterns as the [`.gitignore` specification](https://git-scm.com/docs/gitignore). You should quote your patterns in order to avoid shell interpretation of glob patterns.
+* **Multiple Arguments**: Yes
+* **Default Value**: whatever the default value is OR N/A
```

Comment:
This is TODO?

Reply:
```suggestion
* **Default Value**: None.
```

Reply:
this line shouldn't have been included. sorry! removed